core/model/src/main/resources/bindings.xjb

```diff
@@ -217,9 +212,10 @@
                     if (this.isEmpty()) {
                         return new java.util.ArrayList<>();
                     }
-                    java.util.List<License> allLic = getLeftStatement().getLicenses();
-                    allLic.addAll(getRightStatement().getLicenses());
-                    return allLic;
+                    return java.util.stream.Stream.of(getLeftStatement().getLicenses(), getRightStatement().getLicenses())
```

Comment:
Maybe filter out null entries? I am unsure whether they are possible.
```suggestion
                    return java.util.stream.Stream.of(getLeftStatement().getLicenses(), getRightStatement().getLicenses())
                        .filter(Objects::nonNull)
```

Reply:
We are checking before if left or right statement is null and return an empty list... https://github.com/eclipse/antenna/pull/216/files/aaeec8b839c625a37e1ce6a04e0780f058e92ce4#diff-c6e8267f8d4694964dc7847e6ad515d6R212